test/private/initializeGlobalState.m

```diff
@@ -0,0 +1,47 @@
+function orig = initializeGlobalState()
+% Initialize global state. Set various properties of the graphical root to ensure reliable output of the ACID testsuite.
+% See #542 and #552
+    fprintf('Initialize global state...\n');
+
+    %--- Define desired global state properties
+    % See http://undocumentedmatlab.com/blog/getundoc-get-undocumented-object-properties
+    new.defaultAxesColorOrder.val   =               ...
```

Comment:
I'm not completely sure whether we should set the `defaultAxesColorOrder` in R2014b. I prefer the new colors, and it is a nice check to see the difference between automatically assigned colors (in R2014b) and explicit colors (e.g.`'b'`), and for me it's a visual double check to see the version of the generating version (i.e. R2014b vs R2014a/Octave).

I see what you are trying to do: make sure that identical tests produce identical output regardless of the version where it runs. And that's a noble goal, but I have doubts about the feasibility of this goal. In any case, I think that goal should be addressed later on: first make sure we can get the tests reliable, then uniform.

In general, I would say we should aim for a minimalist approach: only set the properties that make some test go from unreliable to unreliable. Ideally, we should document those cases here (either using the test number or preferably even the actual thing that goes wrong (e.g. for DPI: this interacts with the unit conversions).